config/configgrpc/configgrpc_test.go

```diff
@@ -102,11 +102,10 @@ func TestDefaultGrpcClientSettings(t *testing.T) {
 	tt, err := componenttest.SetupTelemetry(componentID)
 	require.NoError(t, err)
 	t.Cleanup(func() { require.NoError(t, tt.Shutdown(context.Background())) })
-
+	clientConfig := configtls.NewDefaultClientConfig()
+	clientConfig.Insecure = true
 	gcs := &ClientConfig{
```

Comment:
>In most/all of these first cases, wouldn't you want to use the NewDefaultClientConfig at the outer level, rather than only on the inner TLSSetting?

Thanks for taking a look! This suggested change should be done for all the test cases or only the initial ones? PR includes changes only to migrate to new configtls.NewDefaultClientConfig(). If the community is okay, I am willing to create an issue to track this and create separate PR addressing this. 

Please share your thoughts.

Reply:
Please ignore my comment if it doesn't match the requirements.  I just thought it odd that we'd use one and not the other.  I'm new here. :)